lib/engine/config/game/g_18_cz.rb

```diff
@@ -1133,10 +1133,6 @@ module G18CZ
                         0,
                         40
                     ],
```

Comment:
deleting coords from corp configs means that the card looks wonky, since the home coordinates went poof

![Screenshot 2021-01-19 at 8 23 29 PM](https://user-images.githubusercontent.com/1711810/105126810-63259880-5a94-11eb-9c85-3f4375701e9d.png)

Reply:
check this commit out which uses coordinate lookups instead of hardcoding

https://github.com/benjaminxscott/18xx/commit/bedbef8021ef8f2d704940a5f148179059065cc0

Reply:
I rewrote the placing of home_tile to use the coordinates direct. Thanks for the info that they are important